### ISSUE TITLE
Make sorting more universally consistent

### ISSUE THREAD

**Issue by DinoInNameOnly** [themotte/rDrama#373]
In the 10/10 culture war thread, [when comments are sorted by top](https://www.themotte.org/post/120/culture-war-roundup-for-the-week?sort=top), KulakRevolt's +69 / -22 [comment](https://www.themotte.org/post/120/culture-war-roundup-for-the-week/16741?context=8#context) shows up above freemcflurry's +60 / -4 [comment](https://www.themotte.org/post/120/culture-war-roundup-for-the-week/16064?context=8#context). My speculation is that this is because the sorting is based only on upvotes and ignores downvotes.

**Comment by zorbathut** [comment]
Yeah, this is a general problem we have all over the place. We really do need to do a pass over this to make sure sorting is actually happening properly in every place. I've updated this as appropriate.
